.github/workflows/tests.yml

```diff
@@ -51,8 +51,9 @@ jobs:
 
       - name: Install odxtools development and testing dependencies
         run: |
-          pip install -r requirements.txt
-          pip install -r test-requirements.txt
+          pip install --upgrade pip
+          pip install -e .
```

Comment:
don't use editable mode in CI, you will miss bugs such as  #179 if you do

Reply:
okay. As you've probably noticed, I'm not familiar with this new-fanged `pyproject.toml` stuff, so I used the cantools files as a template. Anyway, it is good to follow the [official advise](https://pip.pypa.io/en/stable/topics/local-project-installs/#regular-installs), thanks for pointing this out!